data/data/coreos/rhcos.json

```diff
@@ -1,83 +1,92 @@
 {
   "stream": "rhcos-4.10",
   "metadata": {
-    "last-modified": "2021-11-11T19:40:38Z",
-    "generator": "plume cosa2stream 0.11.0+454-g226dbc8e9"
+    "last-modified": "2021-12-06T18:03:33Z",
+    "generator": "plume cosa2stream 0.12.0+47-g052fabbb7"
   },
   "architectures": {
     "aarch64": {
       "artifacts": {
         "aws": {
-          "release": "410.84.202111111403-0",
+          "release": "410.84.202112060851-0",
           "formats": {
             "vmdk.gz": {
               "disk": {
-                "location": "https://rhcos-redirector.apps.art.xq1c.p1.openshiftapps.com/art/storage/releases/rhcos-4.10-aarch64/410.84.202111111403-0/aarch64/rhcos-410.84.202111111403-0-aws.aarch64.vmdk.gz",
-                "sha256": "b355cb94d0e5e640f729419eea34f0a5fbd2dcb181bb00b229fcef2892bef660",
-                "uncompressed-sha256": "cd33b106831eec1dbb2914bec9df6b84b5e821ef639a35195e2681512530ddca"
+                "location": "https://rhcos-redirector.apps.art.xq1c.p1.openshiftapps.com/art/storage/releases/rhcos-4.10-aarch64/410.84.202112060851-0/aarch64/rhcos-410.84.202112060851-0-aws.aarch64.vmdk.gz",
+                "signature": "https://rhcos-redirector.apps.art.xq1c.p1.openshiftapps.com/art/storage/releases/rhcos-4.10-aarch64/410.84.202112060851-0/aarch64/rhcos-410.84.202112060851-0-aws.aarch64.vmdk.gz.sig",
```

Comment:
This is a new field for all the artifacts that we have not used in RHCOS before. If one were to follow this link, it would return an error.

This looks like it could be fallout related to https://github.com/coreos/stream-metadata-go/pull/33

cc: @bgilbert

Reply:
Oh wait, we have to pass `--no-signatures`; see https://github.com/openshift/installer/pull/5268/

@saqibali-2k looks like you need to rerun the `plume` command